File: Logic_HandsOn/C/Problems on Digits,String,Array,Pointer etc/program96.c
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function name 	: CountCapital
// Input	 	: character
// Output	 	: integer
// Description   	: It is Used return number of capital characters.
// Author	 	: Vishal Tejkant Patil
// Date		: 25/09/2021
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>
#include<stdbool.h>

int CountCapital(char *Brr)
{
	int iCnt=0;
	while(*Brr!='\0')
	{
		if(*Brr>='A' && *Brr<='Z')
		{
			iCnt++;
		}
		Brr++;
	}

	return iCnt;
}

int main()
{
	char Arr[30];
	int iRet=0;

	printf("Enter Character: ");
	scanf("%[^'\n']s",Arr);

	iRet = CountCapital(Arr);

	printf("Capital are :%d\n",iRet);

	return 0;
}

/*
OUTPUT:
Enter Character: Marvellous Multi OS
Capital are :4


*/

File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program96.java
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//		Program:
//			Write a program which accept string from user and count number of capital characters.			
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Solution
{
	String str;
	int iCnt,iCount;

	Solution(String str)
	{
		this.str=str;
		this.iCnt=0;
		this.iCount=0;
	}

	int CountCapital()
	{
		
		while(iCnt!=str.length())
		{
			if(str.charAt(iCnt)>='A' && str.charAt(iCnt)<='Z')
			{
				iCount++;
			}
			iCnt++;
		}
		
		return iCount;
	}
}

class Program
{
	public static void main(String args[])
	{
		int iRet=0;
		Scanner sObj = new Scanner(System.in);
		System.out.println("Enter the string: ");
		String str = sObj.nextLine();

		Solution obj = new Solution(str);
		iRet = obj.CountCapital();

		System.out.println("Capital are: "+iRet);
	}
}

/*
OUTPUT:
Enter the string: 
Vishal PAtil
Capital are: 3

*/
